Fabricator.Resources/HostFile.fs

```diff
@@ -0,0 +1,141 @@
+// SPDX-FileCopyrightText: 2020-2025 Friedrich von Never <friedrich@fornever.me>
+//
+// SPDX-License-Identifier: MIT
+
+namespace Fabricator.Resources
+
+open System
+open System.IO
+open System.Runtime.InteropServices
+open Fabricator.Core
+open TruePath
+open TruePath.SystemIo
+
+type HostFile =
+    /// <summary>
+    /// Gets the default hosts file path for the current platform.
+    /// </summary>
+    static member internal DefaultHostsPath: AbsolutePath =
+        if RuntimeInformation.IsOSPlatform(OSPlatform.Windows) then
+            AbsolutePath @"C:\Windows\System32\drivers\etc\hosts"
```

Comment:
Use `SpecialFolder.System` instead of hardcode.

In tests, obtain it from environment - thus leave tests portable to non-standard Windows installations, but still check if this will do the correct thing.